ReactiveExtensions/UIKit/UIStepper.swift

```diff
@@ -36,12 +37,26 @@ public extension Rac where Object: UIStepper {
     }
   }
 
-  var maximumValue: Signal<Double, Never> {
+  var stepValue: Signal<Double, Never> {
     nonmutating set {
       let prop: MutableProperty<Double> = lazyMutableProperty(
-        object, key: &Associations.maximumValue,
-        setter: { [weak object] in object?.maximumValue = $0 },
-        getter: { [weak object] in object?.maximumValue ?? 100.0 })
+        object, key: &Associations.stepValue,
+        setter: { [weak object] in object?.stepValue = ($0 > 0 ? $0 : 1.0 ) },
```

Comment:
Based on the [documentation](https://developer.apple.com/documentation/uikit/uistepper/1624083-stepvalue), `UIStepper` would raise ___ if `stepValue` was set to `0` or `negative value` so here's is a small sanitation in order for us not to make `UIKit` crash.